pages/index.html

```diff
@@ -1,12 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Esther's Portfolio</title>
+    <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon">
+    <link href="/styles/style.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
 </head>
+
 <body>
-  
+    <header class="header-container">
+      <h1 id="page-title">Esther's Portfolio</h1>
+      <nav class="nav">
+        <ul>
+          <li><a href="index.html">Home</a> |</li>
+          <li><a href="about.html">About Me</a> |</li>
+          <li><a href="portfolio.html">Portfolio</a> |</li>
+          <li><a href="contact.html">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      
+      <div class="intro-container">
+        <section>
```

Comment:
`section` is great when we have multiple chunks of content that are children of the same parent but that we want to treat separately and show the distinction between through HTML. Since there is only one `section` inside `main` I would try seeing if the `div` and maybe even the `section` could elements could be removed, nesting everything as direct children of `main` and giving it the `intro-container` tag for styling.